src/products/firefox_desktop/firefox_l10n_faqs.md

```diff
@@ -125,6 +125,15 @@ Local builds:
 * You can [build directly in a different locale](https://firefox-source-docs.mozilla.org/build/buildsystem/locales.html#instructions-for-single-locale-repacks-for-developers) (no support for artifact builds).
 * It’s possible to add specific l10n repacks to a [Try Server](https://wiki.mozilla.org/ReleaseEngineering/TryServer#Scheduling_jobs_with_Treeherder) push. For example, to add French: select *Add new Jobs*, then select the `fr` locale in `L10n-Rpk`. Once the job is completed, builds will be available as artifacts under `L10n (BsX)`, where `X` changes depending on the locale.
 
+### Can I land strings early, before my feature is fully built-out?
+
+This makes it difficult for our localizers to know how the strings are being used, which can make it harder to supply a translation that fully fits the context. If you plan on going this route, it is only advisable with the following conditions:
```

Comment:
```suggestion
Except for special cases, strings should be landed at the same time as the associated UI changes. Otherwise, this makes it difficult for our localizers to know how the strings are being used, which can make it harder to supply an accurate translation that fully fits the context. 

Please consult with a [localization PM](#who-can-i-contact-if-i-have-more-questions) if you have a need to land strings early. Note that landing strings early is only advisable with the following conditions:
```